build_tools/ci/run_matmul_test.sh

```diff
@@ -39,6 +56,13 @@ TEST_RUNNER="${IREE_INSTALL_BIN}/iree-e2e-matmul-test"
 echo "Python version: $("${IREE_PYTHON3_EXECUTABLE}" --version)"
 echo "iree-compile version: $("${IREE_COMPILE_EXE}" --version)"
 mkdir -p ${OUTPUT_DIR}
+cd ${OUTPUT_DIR}
+
+python3 -m venv sandbox
+source sandbox/bin/activate
```

Comment:
Does iree-amd-aie have an established naming convention for venv directories? In IREE we suffix them with `.venv` (just `.venv` on its own with no prefix is the default used in many places)